doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,396 @@
+
+The water volume fraction of the wet sand used in the experiment is 0.325 %. According to the work of Liefferink *et al.* [#Liefferink2018]_, this water volume fraction matches an ``friction coefficient`` of 0.5. We use this value as a rough estimate of the ``friction coefficient`` for this example.
+
+.. code-block:: text
+
+  subsection lagrangian physical properties
+  set g                        = -9.81, 0, 0
+  set number of particle types = 1
+  subsection particle type 0
+  set size distribution type            = custom
+  set custom diameters                  = 1.02e-4 , 1.16e-4 , 1.33e-4 , 1.52e-4 , 1.75e-4 , 2.00e-4 , 2.29e-4 , 2.62e-4 , 3.01e-4 , 3.44e-4
+  set custom volume fractions           = 0.04 , 0.06 , 0.07 , 0.10 , 0.13 , 0.15 , 0.16 , 0.14 , 0.10 , 0.05
+  set number of particles               = 200000
+  set density particles                 = 1922
+  set young modulus particles           = 5.94e4
+  set poisson ratio particles           = 0.4
+  set restitution coefficient particles = 0.9
+  set friction coefficient particles    = 0.5
+  set rolling friction particles        = 0.5
+  set surface energy particles          = 0.01
+  end
+  set young modulus wall           = 1e7
+  set poisson ratio wall           = 0.33
+  set restitution coefficient wall = 0.9
+  set friction coefficient wall    = 0.5
+  set rolling friction wall        = 0.5
+  set surface energy wall          = 1e-4
+  end
+
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~~~~~
+ 
+
+The JKR contact model is used in this case because it has be shown to correctly model particle agglomeration for wet sand [#hu2022]_ .
+
+.. code-block:: text
+
+  subsection model parameters
+  subsection contact detection
+  set contact detection method                = dynamic
+  set dynamic contact search size coefficient = 0.9
+  set neighborhood threshold                  = 1.3
+  end
+  subsection load balancing
+  set load balance method = frequent
+  set frequency           = 100000
+  end
+  set particle particle contact force method = hertz_JKR
+  set rolling resistance torque method       = constant_resistance
+  set particle wall contact force method     = JKR
+  set integration method                     = velocity_verlet
+  end
+
+  subsection restart
+  set checkpoint = true
+  set frequency  = 10000
+  set restart    = false
+  set filename   = dem
+  end
+
+Particle Insertion
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylinder. 10 000 particles are inserted every 9000 iterations. It is important to note the size of the insertion box to ensure it is completely inside our cylinder which is smaller than our domain. Otherwise, particles will be lost during the insertion stage given that the domain's bottom is an outlet.
```

Comment:
```suggestion
An insertion box is defined inside the cylinder. 10 000 particles are inserted every 9000 iterations. The size of the insertion box is chosen to ensure it is completely inside our cylinder which is smaller than our domain. Otherwise, particles will be lost during the insertion stage.
```